dts/arm/nordic/nrf9160ns.dtsi

```diff
@@ -0,0 +1,71 @@
+#include <arm/armv8-m.dtsi>
+#include <dt-bindings/i2c/i2c.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m33f";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+	};
+
+	aliases {
+		rtc-1 = &rtc1;
+		clock = &clock;
+		power = &power;
+		nvmc = &nvmc;
+		uart-0 = &uart0;
+		uart-1 = &uart1;
+		uart-2 = &uart2;
+		uart-3 = &uart3;
+		adc-0 = &adc;
+		gpio-0 = &gpio0;
+		gpiote-0 = &gpiote; /* FIXME alias should be gpiote-1 */
```

Comment:
why the FIXME?

Reply:
Because in nRF9160 GPIOTE0 is fixed-Secure, while GPIOTE1 is fixed-NonSecure.
So, ideally, Non-Secure dtsi should use gpiote-1.
FIXME is added, so we remember to do this once GPIOTE driver supports dual instances.